starter-code/Bookshelf/app/src/main/java/ly/generalassemb/drewmahrt/bookshelf/MainActivity.java

```diff
@@ -52,11 +58,37 @@ public long getItemId(int position) {
             @Override
             public View getView(int position, View convertView, ViewGroup parent) {
                 //TODO: Update the view
+                View cView = convertView;
+                if(convertView == null) {
+                    LayoutInflater inflater = LayoutInflater.from(MainActivity.this);
+                    cView = inflater.inflate(android.R.layout.simple_list_item_1, null);
+                }
+                TextView textView = (TextView)cView.findViewById(android.R.id.text1);
+                textView.setText(mBookList.get(position).getTitle() + "\n"
+                +mBookList.get(position).getAuthor());
```

Comment:
When one statement wraps onto two lines, it's a good idea to indent the second line to indicate it's a continuation of the first.